airbyte-integrations/connectors/destination-iceberg/src/test-integration/java/io/airbyte/integrations/destination/iceberg/glue/BaseIcebergGlueCatalogS3IntegrationTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.iceberg.glue;
+
+import static io.airbyte.integrations.destination.iceberg.IcebergIntegrationTestUtil.ICEBERG_IMAGE_NAME;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.airbyte.cdk.integrations.standardtest.destination.DestinationAcceptanceTest;
+import io.airbyte.commons.io.IOs;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.integrations.destination.iceberg.IcebergIntegrationTestUtil;
+import io.airbyte.integrations.destination.iceberg.config.catalog.IcebergCatalogConfig;
+import io.airbyte.integrations.destination.iceberg.config.catalog.IcebergCatalogConfigFactory;
+import io.airbyte.integrations.destination.iceberg.config.format.DataFileFormat;
+import java.nio.file.Path;
+import java.util.HashSet;
+import java.util.List;
+import org.apache.commons.lang.RandomStringUtils;
+import org.apache.iceberg.aws.glue.GlueCatalog;
+import org.apache.iceberg.catalog.Namespace;
+import org.apache.iceberg.catalog.TableIdentifier;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+public abstract class BaseIcebergGlueCatalogS3IntegrationTest extends DestinationAcceptanceTest {
+
+  private JsonNode config;
+  private GlueCatalog glueCatalog;
+  private String database;
+
+  public abstract DataFileFormat getFormat();
+
+  @Override
+  protected void setup(final @NotNull TestDestinationEnv testEnv, final @NotNull HashSet<String> TEST_SCHEMAS) {
+    final JsonNode baseConfig = Jsons.deserialize(IOs.readFile(Path.of("secrets/s3_glue_config.json")));
+    database = "ab_iceberg_tests_" + RandomStringUtils.randomAlphabetic(4).toLowerCase();
+    config = Jsons.clone(baseConfig);
+    final JsonNode formatConfigJson = config.get("format_config");
+    if (formatConfigJson instanceof ObjectNode) {
```

Comment:
nit: we control the contents of `s3_glue_config.json`, can we just assume it's always an objectnode + hard fail if it's not?